app-config-node/src/prompts.test.ts

```diff
@@ -37,8 +37,7 @@ describe('consumeStdin', () => {
         .then(() => end())
         .catch(() => {});
 
-      // we expect newlines to be eaten up, since this function is used for html and base64 data
```

Comment:
this explains it eh? Should `app-config decrypt` swallow newlines?

Reply:
I would say no, decrypt should not. TBH not sure what that even means if you have an encrypted value with multiple lines.

Reply:
Do you mind expanding on the html and base64 use cases?

Reply:
HTML makes no sense, but I could see it being possible that base64 values might be split up? tbh it sounds like something I ran into while between different formats of encrypted values, some of which in early prototypes were split up iirc.

So I guess answer here is let's ignore it - if no tests fail, then it's a :+1: for me